src/tools.rs

```diff
@@ -113,6 +121,13 @@ impl Application {
               _ => bail!("Unable to download Sass for {target_os} {target_arch}")
             },
 
+            Self::TailwindCss => match (target_os, target_arch) {
+                ("windows", "x86_64") => format!("https://github.com/tailwindlabs/tailwindcss/releases/download/{version}/tailwindcss-windows-x64.exe"),
+                ("macos" | "linux", "x86_64") => format!("https://github.com/tailwindlabs/tailwindcss/releases/download/{version}/tailwindcss-{target_os}-x64"),
+                ("macos" | "linux", "aarch64") => format!("https://github.com/tailwindlabs/tailwindcss/releases/download/{version}/tailwindcss-{target_os}-arm64"),
+                _ => bail!("Unable to download tailwindcss for {target_os} {target_arch}")
+            },
```

Comment:
We should at least add the linux x64 & arm64 binaries as well, given that the release artifacts are available and we also support these targets for the other tools which we have here.

Reply:
I think it does. I don't have a system handy to test it but I'll let you know once macos works and we can decide what to do from there.